block-padding/src/lib.rs

```diff
@@ -116,9 +116,8 @@ impl<B: ArrayLength<u8>> Padding<B> for Pkcs7 {
             *b = n;
         }
     }
-
     #[inline]
-    fn unpad(block: &Block<B>) -> Result<&[u8], UnpadError> {
+    fn unpad<B: ArrayLength<u8>>(block: &Block<B>, strict: bool) -> Result<&[u8], UnpadError> {
```

Comment:
So the `strict` flag is the main reason for changing this block to inherent methods?

If so, it only seems necessary for `unpad`, not `pad`.